pages/linux/nautilus.md

```diff
@@ -0,0 +1,13 @@
+# nautilus
+
+> Default file explorer for Gnome desktop environment.
+> Also known as Gnome Files.
+> More information: <https://github.com/GNOME/nautilus>.
```

Comment:
```suggestion
> More information: <https://manned.org/nautilus>.
```
Altough the Github repo is more official, it doesn't give any information on how to use `nautilus` in the command line. A man page is problably more helpful if you want to find out more about the command.

Reply:
good to know. i didn't know if the 'more info' was for more info on the full program, or just the terminal command. sounds like it's the terminal command, so i'll keep that in mind going forward.

Reply:
It depends. If the official page covers all of the command line options it should be preffered. Otherwise we use `manned.org` or a distro specific man page, if the command is distro specific.